src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -145,9 +147,36 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $io->listing($items);
         }
 
+        $list = array();
+        foreach ($this->getLoaderPaths() as $namespace => $paths) {
+            $list = array_merge($list, array_map(function ($path) use ($namespace) {
+                return $namespace.($namespace ? ': ' : '').$path;
```

Comment:
What about substr the project dir if provided as a constructor argument of the command?

```php
if (null !== $this->projectDir && 0 === strpos($path, $this->projectDir)) {
    $path = ltrim(substr($file, strlen($this->projectDir)), DIRECTORY_SEPARATOR);
}
```

Reply:
Nice idea! Done.